File: 52a/main.go
package main

import (
	"fmt"
)

func main() {
  var length int
  var width int
	fmt.Println("length: ")
  fmt.Scan(&length)
  fmt.Println("width: ")
  fmt.Scan(&width)

  var area int
  area = length * width
  var perimeter int
  perimeter = (length * 2) + (width * 2)
  fmt.Println("area:", area)
  fmt.Println("perimeter:", perimeter)
  
}

/*
length:
143
width:
82
area: 11726
perimeter: 450
*/

File: 52a/main.cpp
#include <iostream>
using namespace std; 

int main() {
  int length;
  cout << "length: ";
  cin >> length;

  int width;
  cout << "width: ";
  cin >> width;

  double area = length * width;
  double perimeter = length * 2 + width * 2;

  cout << "area is: " << area;
  cout << "\nperimeter is: " << perimeter;
  
  return 0;
}

/*
length: 143
width: 82
area is: 11726
perimeter is: 450
*/